config.example.json

```diff
@@ -1,13 +1,16 @@
 {
-  "name": "Discord.JS Boilerplate",
-  "botToken": "xxx",
-  "prefix": "!",
-  "prefixSeperated": false,
+  "name": "Testing Bot",
+  "botToken": "your token here!",
+  "prefix": "nyan!",
+  "prefixSeperated": true,
   "channels": {
-    "general": "1234"
+    "approved": ["853572801594916874",
+                 "742679158093643828",
+                 "928666008061882429"
```

Comment:
Oh btw which channels are these?

Reply:
Oh right... why not give each channel a name instead of keeping them in an array ("approved")?
Is this to hide the channel names from public view?

Reply:
These are my own channels in my own testing server so that I don't clog up the main NUSGDG server. Also the approved array shows which channel the bot can communicate in which can be expanded and reduced at any time